### PR TITLE
feat : responsiveness-added

### PR DESCRIPTION
**Pull request by Jia-A**
Added responsiveness

### CODE REVIEW

File: src/dot-components/Cart/cart.css [hunk 1]
```diff
@@ -1,19 +1,18 @@
-
 .cart-container {
     display: flex;
     margin:auto;
     justify-content: center;
     gap: 5rem; 
     flex-wrap: wrap;
-    padding-bottom: 2rem;
+    /* padding-bottom: 2rem; */
```
Comment (zoyatanweer) [inline src/dot-components/Cart/cart.css:7]: remove this

File: src/dot-components/Cart/cart.css [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 }
 
 .cart-cards-container {
     gap: 1.875rem;
     margin-bottom: 2.5rem;
     display: flex;
     flex-wrap: wrap;
-    width: 70%;
+    /* width: 70%; */
```
Comment (zoyatanweer) [inline src/dot-components/Cart/cart.css:15]: remove this

File: src/dot-components/Cart/cart.css [hunk 3]
```diff
@@ -22,14 +21,15 @@
     box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
     padding: var(--content-padding);
     flex-direction: column;
+    align-self: auto;
     background-color: #b8c6d6;
```
Comment (zoyatanweer) [inline src/dot-components/Cart/cart.css:25]: use var for color

File: src/dot-components/Cart/cart.css [hunk 4]
```diff
@@ -22,14 +21,15 @@
...
 }
 
 .cart-head {
     font-weight: 500;
     text-align: center;
     margin: 1.5rem;
-    color: var(--secondary-color);
+    /* color: var(--secondary-color); */
```
Comment (zoyatanweer) [inline src/dot-components/Cart/cart.css:32]: remove commented code

File: src/dot-components/Product-listing/product.css [hunk 5]
```diff
@@ -2,6 +2,7 @@
     display: flex;
     gap: var(--content-padding);
     padding-bottom: 2.5rem;
+    /* justify-content: center; */
```
Comment (zoyatanweer) [inline src/dot-components/Product-listing/product.css:5]: remove if not needed

File: src/dot-components/Product-listing/product.jsx [hunk 6]
```diff
@@ -4,9 +4,10 @@ import { useEffect } from "react";
 import { Link } from "react-router-dom";
 import { Filter } from "../Filter/filter";
 import { useFilter } from "../../Context/filterContext";
-import { Navbar } from "../Navbar/navbar";
+// import { Navbar } from "../Navbar/navbar";
```
Comment (zoyatanweer) [inline src/dot-components/Product-listing/product.jsx:7]: remove commented code

File: src/dot-components/Product-listing/product.jsx [hunk 7]
```diff
@@ -84,9 +85,12 @@ return showProduct.filter(
     };
     return (
     <div className="App">
-        <Navbar />
+        {/* <Navbar /> */}
```
Comment (zoyatanweer) [inline src/dot-components/Product-listing/product.jsx:88]: remove comments

**Review by zoyatanweer — APPROVED** [review]
- UI is awesome! 
- Code quality is good, left a few suggestions and make those changes.
- Good to merge.

**Review by MaruthiPothuganti — APPROVED** [review]
Everything looks good. PR looks good to merge.

File: src/dot-components/Product-listing/product.jsx (hunk 7 above)
Comment (Jia-A) [reply src/dot-components/Product-listing/product.jsx:88]: Yes, will remove all the comments

**Pull request merged**